### PR TITLE
drawpile: init at 2.0.10

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

There's only one issue that I found: I can't get the icons to show up when I run ./result/bin/drawpile . Maybe it's because i use a window manager instead of a full de?

Edit: wait... once the package is installed with nix-env -i the icons show up. Is this normal?

### CODE REVIEW

File: pkgs/applications/graphics/drawpile/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    libmicrohttpd # HTTP admin api
+    giflib # gif animation export support
+    miniupnpc # automatic port forwarding
+    kdnssd # local server discovery with Zeroconf
+  ];
+  configurePhase = "cmake -DCMAKE_INSTALL_PREFIX=$out .";
+
+  meta = with stdenv.lib; {
+    description = "A Free software collaborative drawing program that allows multiple users to sketch on the same canvas simultaneously";
+    homepage = "https://drawpile.net/";
```
Comment (nlewo) [inline pkgs/applications/graphics/drawpile/default.nix]: Do not use `"` for urls

File: pkgs/applications/graphics/drawpile/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    downloadPage = "https://drawpile.net/download/";
```
Comment (nlewo) [inline pkgs/applications/graphics/drawpile/default.nix]: Do not use `"` for urls

File: pkgs/applications/graphics/drawpile/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    libsodium # ext-auth support
+    libmicrohttpd # HTTP admin api
+    giflib # gif animation export support
+    miniupnpc # automatic port forwarding
+    kdnssd # local server discovery with Zeroconf
+  ];
+  configurePhase = "cmake -DCMAKE_INSTALL_PREFIX=$out .";
+
+  meta = with stdenv.lib; {
+    description = "A Free software collaborative drawing program that allows multiple users to sketch on the same canvas simultaneously";
```
Comment (nlewo) [inline pkgs/applications/graphics/drawpile/default.nix]: The licence already says it's a free software:
`'s/Free software //'`

**Review by nlewo — CHANGES_REQUESTED** [review]

**Comment by nlewo** [comment]
@GrahamcOfBorg build drawpile

**Comment by fgaz** [comment]
@nlewo fixed the meta

but I forgot _again_ to rebuild _after_ rebasing on master. I'll fix that.

**Comment by fgaz** [comment]
The build failure is related to cmake 3.11. I opened an issue upstream: https://github.com/drawpile/Drawpile/issues/625

**Comment by nlewo** [comment]
Did you try with `cmake_2_8` (which is available in nixpkgs)?

**Comment by fgaz** [comment]
@nlewo 3.10 is also available and works (`cmake.override {majorVersion="3.10";}`), but I'd rather wait for a release and not pin cmake if you are ok with that

**Comment by nlewo** [comment]
@fgaz yes, it's better to wait for a release. Ping me when it is done.

**Comment by fgaz** [comment]
@nlewo 2.0.11 got released with the fix. I updated the package

**Comment by fgaz** [comment]
@nlewo can we merge this before the branch-off?

**Comment by nlewo** [comment]
Yes. Sorry for the latency. Thx!

**Review by nlewo — APPROVED** [review]

**Pull request merged**